setup.cfg

```diff
@@ -88,6 +88,8 @@ dev =
     pyright
     pytest-dependency
     pytest_httpserver
+    # Required for testing snippets in docs.
+    mlflow-skinny
```

Comment:
Suggestion: can we do it this way so that any automatic sorting won't separate the dependency from the comment?
```suggestion
    mlflow-skinny  # Required for testing snippets in docs.
```

Reply:
OH YES

Reply:
b00477f

Reply:
Nice :D